src/auth/use-authentication.ts

```diff
@@ -0,0 +1,31 @@
+import { AuthorizationGuard, AuthorizationPayload, instanceToScope, Thread } from '@serlo/authorization'
+import { gql } from 'graphql-request'
+
+import { useGraphqlSwrWithAuth } from '@/api/use-graphql-swr'
+import { useInstanceData } from '@/contexts/instance-context'
+import { Instance } from '@serlo/api'
```

Comment:
TODO: `@serlo/api` leaks quite some authentication types because one of the `export *` now also contains some stuff that is not required for the API. We should probably fix those.

Reply:
Resolved via https://github.com/serlo/api.serlo.org/pull/302.